### PR TITLE
Update limits for Kubelet in monitor-resources test

### PR DESCRIPTION
**Pull request by gmarek**
cc @yujuhong @timstclair

### CODE REVIEW

File: test/e2e/monitor_resources.go [hunk 1]
```diff
@@ -40,8 +40,8 @@ var allowedUsage = resourceUsagePerContainer{
 	},
 	"/kubelet": &containerResourceUsage{
 		CPUUsageInCores:         0.1,
-		MemoryUsageInBytes:      150000000,
-		MemoryWorkingSetInBytes: 150000000,
+		MemoryUsageInBytes:      1000000000,
```
Comment (wojtek-t) [inline test/e2e/monitor_resources.go]: Did we decided that we want to do it?
I thought that we want to change set the limits appropriately according to what was written in:  https://github.com/kubernetes/kubernetes/issues/17774#issuecomment-159705367

So to be honest - I would prefer to hold on with this change.

Comment (gmarek) [inline test/e2e/monitor_resources.go]: Sure, I just wanted to send it out, so I won't forget about it again :)

**Comment by gmarek** [comment]
@wojtek-t I think we should proceed with this change to unbreak the test, and note that this has to be changed when #17774 is fixed.

**Comment by wojtek-t** [comment]
OK - can you please leave a TODO to fix it then?

**Comment by gmarek** [comment]
Done.

**Comment by wojtek-t** [comment]
LGTM

**Comment by gmarek** [comment]
@k8s-bot test this

**Pull request merged**